site/content/blog/2019-10-21-localizing-matterpoll/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Localizing Matterpoll"
+slug: localizing-matterpoll
+date: 2019-11-05T10:49:35+02:00
+categories:
+    - "plugins"
+    - "go"
+author: Ben Schumacher
+github: hanzei
+community: hanzei
+---
+
+[Matterpoll](https://github.com/matterpoll/matterpoll) is a plugin that allows creating polls inside of Mattermost. Given that Mattermost is [localized in 16 different languages](https://docs.mattermost.com/developer/localization.html) we surely want to accomplish the same for Matterpoll. Because we rely on contributors to do the translations we wanted to make sure it's a easy as possible for them to translate new strings and determine if already translated stings needs to be updated because the "source" text changed. On the other hand Matterpoll only has two maintainer ([@kaaka](https://github.com/kaakaa) and [me](https://github.com/hanzei)) and no own infrastructure to work with. Using some translation server like [Transifex](https://en.wikipedia.org/wiki/Transifex) or [Weblate](https://en.wikipedia.org/wiki/Weblate) is not an option.
```

Comment:
```suggestion
[Matterpoll](https://github.com/matterpoll/matterpoll) is a plugin that allows creating polls inside of Mattermost. Given that Mattermost is [localized in 16 different languages](https://docs.mattermost.com/developer/localization.html) we surely want to accomplish the same for Matterpoll. Because we rely on contributors to do the translations we wanted to make sure it's a easy as possible for them to translate new strings and determine if already translated stings needs to be updated because the "source" text changed. On the other hand Matterpoll only has two maintainer ([@kaakaa](https://github.com/kaakaa) and [me](https://github.com/hanzei)) and no own infrastructure to work with. Using some translation server like [Transifex](https://en.wikipedia.org/wiki/Transifex) or [Weblate](https://en.wikipedia.org/wiki/Weblate) is not an option.
```

Let me add a tiny fix about my name.

Reply:
My bad, thanks for the catch :+1: